std/utf.d

```diff
@@ -4270,6 +4270,19 @@ if (isSomeChar!C)
         {
             static struct Result
             {
+                this(return R r)
+                {
+                    this.r = r;
```

Comment:
Okay. I probably need to re-read the copy constructor DIP, but this looks like it's defined to only copy the range being wrapped and not the other members. If so, then how is this range being copied properly? Also, was it the intention to make it equivalent to calling `save`? If so, then it's not, because it doesn't call `save` on the range being wrapped, and what happens then depends entirely on how that particular range is implemented (e.g. if it's a class, it's not being `save`d).